python/neuroglancer/viewer_state.py

```diff
@@ -158,6 +158,12 @@ class BlendTool(Tool):
     TOOL_TYPE = "blend"
 
 
+@export_tool
+class VolumeRenderingTool(Tool):
+    __slots__ = ()
+    TOOL_TYPE = "volumeRendering"
```

Comment:
Per the comment in the other PR, I think there could be one option that controls the volume rendering mode, with one mode being "off" --- then there wouldn't be a separate tool and python option.

Reply:
Agreed, I'll change the other PR to have this new mode controllable from Python and remove the new tool from this PR